api/src/main/java/io/strimzi/api/kafka/model/KafkaRebalanceSpec.java

```diff
@@ -62,6 +63,17 @@ public void setSkipHardGoalCheck(boolean skipHardGoalCheck) {
         this.skipHardGoalCheck = skipHardGoalCheck;
     }
 
+    @Description("Whether to balance load between disks within brokers, known as intra-broker disk balancing (requires JBOD Kafka deployment. ) " +
```

Comment:
I did not noticed it before ... but what does `load` stand for here? Is it load as in amount of IOPS? Or load as in disk space utilization? Or both? Might be good to clarify this.

Reply:
Load stand for disk space utilization here, so I have update the description!